cg/meta/upload/scout/mip_config_builder.py

```diff
@@ -36,7 +33,9 @@ def __init__(
         )
         self.case_tags: CaseTags = CaseTags(**MIP_CASE_TAGS)
         self.sample_tags: SampleTags = SampleTags(**MIP_SAMPLE_TAGS)
-        self.load_config: MipLoadConfig = MipLoadConfig(track="rare")
+        self.load_config: MipLoadConfig = MipLoadConfig(
+            track="rare", delivery_report=self.get_file_from_hk({HK_DELIVERY_REPORT_TAG})
```

Comment:
rare what? Would be nice with a more descriptive name.

Reply:
It's defined as `Literal["rare", "cancer"]` in Scout but I've created some constants for the tracks, good catch 👍